packages/api/tests/sale.test.js

```diff
@@ -388,6 +388,7 @@ describe('Sale Tests', () => {
             crop_variety_id: cropVariety2.crop_variety_id,
           },
         ],
+        revenue_type_id: 1,
```

Comment:
should this be added to mocks.fakeSale()? Or maybe not unless renamed fakeCropSale()..

Also to above point about  setting type id as 1 -- manually setting type id versus retrieving it.. this style will only work for this one default sale type if we add CSA box or Processed or something in the future it could be a bad pattern.

Reply:
I'll add a comment. This is a placeholder to not crash the Jest tests; I don't think it makes any sense to touch fakeSale() now, but rather it needs to be updated along with the sale controller (and flow) when we actually have a way to specify which revenue type is associated with a sale. Obviously we do not want to hardcode `1` at that point!

Reply:
Maybe a note on the Jira ticket?

Reply:
That's what I was thinking -- do you know which one aligns closest with integrating revenue type into sales?

Reply:
APIs? since it requests jest tests? LF-3584